src/components/overlay/editors/text.tsx

```diff
@@ -25,7 +25,9 @@ export class TextEditor implements Edition.EditorBase {
 
     private onKeyDown(e: KeyboardEvent): void {
         const isEnter: boolean = codesLetter.ENTER === e.code;
-        if (isEnter && e.target && this.editCallback) {
+        const isTab: boolean = codesLetter.TAB === e.code;
```

Comment:
For the future it's better to collect keys check in separate service: keyCodes.ts and keyCodes.utils.ts